both/api/source-imports/server/stats.js

```diff
@@ -22,42 +22,60 @@ SourceImports.helpers({
   generateAndSaveStats() {
     const attributeDistribution = {};
 
-    function explore(rootKey, valueOrAttributes) {
-      if (_.get(blacklist, rootKey) === true) {
+    // Goes through all attributes of a `PlaceInfo` object recursively
+    // and increments the according values in `attributeDistribution` to calculate
+    // each attribute value's frequency in the whole dataset.
+    function exploreAttributesTree(rootKey, valueOrAttributes) {
```

Comment:
Just a thought on the side - I wonder if we could move `exploreAttributesTree` to a different file and unit test it? Only saying so as it looks like the kind of logic that would benefit from such a test. Not necessarily one for this PR, but id it makes sense this could be added to the backlog.